### PR TITLE
Patch requirements.txt

### PR DESCRIPTION
**Pull request by ZenulAbidin**
Currently the `docker build`, `pip install -r requirements.txt` and `python run main.py` commands all fail with the current requirements.txt due to deprecated or removed APIs, so I pinned older versions of nested dependencies inside the file.

In particular:

- A new version of `itsdangerous` is being pulled which [removed](https://serverfault.com/questions/1094062/error-from-itsdangerous-import-json-as-json-importerror-cannot-import-name-j) the `json` submodule. This causes runtime failure and breaks Docker build
- `argostranslate` and `ctranslate2` had to be downgraded to an older version and inserted respectively for the pip install process to succeed.

### CODE REVIEW

**Comment by pierotofy** [comment]
Hey @ZenulAbidin :hand: thanks for the patch, but I'm not sure I understand the need to downgrade argostranslate, the current version built and pushed correctly to docker hub just a few days ago?

**Comment by ZenulAbidin** [comment]
@pierotofy I downgraded argostranslate just in case it caused dependency errors with ctranslate2, I will try again with the original argostranslate version and report the results.

**Comment by ZenulAbidin** [comment]
@pierotofy Confirmed. It works with argostranslate 1.6.1, the original version.

**Comment by dingedi** [comment]
I think you should not specify a version for ctranslate2 because it is specified by argostranslate, this will avoid problems.

File: requirements.txt [hunk 1]
```diff
@@ -1,6 +1,6 @@
-ctranslate2==1.16.2
+ctranslate2==2.10.1
```
Comment (dingedi) [inline requirements.txt:1]: I think you should not specify a version for ctranslate2 because it is specified by argostranslate, this will avoid problems.

Comment (ZenulAbidin) [reply requirements.txt:1]: I inserted this line before updating `pip`, I'll go check if it works without it on the newer version of pip.

It's possible that this PR may not even be necessary if `pip install` works without modifications to requirements.txt - we just need to tell people in the README to update their `pip` to the latest version before installing from source (virtualenvs, unfortunately, do not install newest pip when creating the environment).

**Comment by ZenulAbidin** [comment]
It works unmodified with updated pip too. Closing this PR.

**Pull request closed without merging**